src/components/pages/edit/edit.vue

```diff
@@ -106,7 +120,7 @@
           description: '',
           thumbnail: '',
           contentCategory: '',
-          licenses: [],
+          licenses: ["MIT bUlLShIt"],
```

Comment:
Eventuell, anderen Placeholder ^^
```suggestion
          licenses: ["Test Licence"],
```